### ISSUE TITLE
Default font changed for results grid

### ISSUE THREAD

**Issue by kburtram** [Microsoft/sqlopsstudio#986]
- SQL Operations Studio Version: 0.27.2

Steps to Reproduce:
1. Run a query

Expected: Default font is same as 0.27.1
Actual: Default font is different

<img width="818" alt="screen shot 2018-03-26 at 12 42 20 pm" src="https://user-images.githubusercontent.com/599935/37928899-3dab61d4-30f3-11e8-8e4f-d8c636377e76.png">

**Comment by kburtram** [comment]
@anthonydresser I think this is likely related to the grid options fix we merged yesterday.  Is it easy to get the default font to be the same as before, or should I revert that change?

**Comment by kburtram** [comment]
Also, it looks like the Grid font settings are also changing the tab font settings.

<img width="677" alt="screen shot 2018-03-26 at 12 46 02 pm" src="https://user-images.githubusercontent.com/599935/37929054-b5944f08-30f3-11e8-9ffa-eb0b1ed8b250.png">

**Comment by anthonydresser** [comment]
It uses the same default font as the editor.

**Comment by kburtram** [comment]
If we plan to change the default font from what we've previously shipped then we'll need to go through a UX review process.  This isn't the type of change we can slip in through a bug fix, but needs to be an explicit design change.

I thought we already went through a similar process with the vscode-mssql extension, and we got it right there IIRC.  I don't remember the exact details, but I think we ended up with separate font settings for editor and result grid, similar to what we support in SSMS.

If there is an easy fix then let's do that.  Otherwise, we should rollback this change until the UX changes are reviewed, or we confirm we aren't significantly changing the UX we previously shipped.
